lib/routes/components.js

```diff
@@ -83,6 +83,20 @@ module.exports = app => {
 				return response.redirect(307, `${request.basePath}components/${component.name}@${component.version}`);
 			}
 
+			// Load dependencies
+			let dependencies;
+			if (component.resources.demos) {
```

Comment:
is this supposed to be `component.resources.dependencies`?

Reply:
😬🍝